fighthealthinsurance/ml/ml_models.py

```
@@ -252,6 +252,15 @@ def is_professional_tone(self, result: Optional[str]) -> bool:
             return False
         return True
 
+    def is_professional_tone(self, result: Optional[str]) -> bool:
```

Ruff (0.8.2)

Redefinition of unused `is_professional_tone` from line 246

(F811)